samples/maps/demo/globe-in-variable-pie/demo.js

```diff
@@ -0,0 +1,586 @@
+                        pointData.population,
+                        pointData.metricTons,
+                        pointData.ouncesPerCapita
+                    ]
+                    ),
+                    colors:
+                    [
+                        '#ccb339',
+                        '#d2ba45',
+                        '#d7c150',
+                        '#ddc85c',
+                        '#e3ce68',
+                        '#e8d673',
+                        '#eedc7f',
+                        '#f4e38b',
+                        '#f9ea96',
+                        '#fff1a2'
+                    ],
+                    point: {
+                        events: {
+                            unselect() {
+                                const point = this,
+                                    countryName = point.name,
+                                    mapPoint =
+                                        chart.findCountry(countryName);
+                                mapPoint.update({
+                                    color: null
+                                });
+
+                                if (countryName === chart.selectedCountry) {
+                                    chart.sticky = chart.sticky?.destroy();
+                                }
+                            },
+                            select() {
+                                const point = this,
+                                    selectedPoints = chart.getSelectedPoints(),
+                                    countryName = point.name,
+                                    mapPoint =
+                                            chart.findCountry(countryName);
+                                mapPoint.update({
+                                    color: '#ffffff99'
+                                });
+                                chart.selectedCountry = countryName;
+                                chart.justSelectedPoint = true;
+
+                                // Remove previously selected points
+                                // from selectedPoints array
+                                selectedPoints.forEach(point => {
+                                    if (point.name !== countryName) {
+                                        point.select(false);
+                                    }
+                                });
+
+                                chart.redraw({
+                                    defer: 500,
+                                    complete() {
+                                        // Circumvent "rogue" firing of this
+                                        // function by only initiating globe
+                                        // rotation if globe is not rotating
+                                        if (!chart.isRotating) {
+                                            chart.rotateToCountry(countryName);
+                                        }
+                                    }
```

Comment:
Maybe this whole defer and check for `chart.isRotating` could be omitted. The animation in Highcharts has built-in functionality to avoid animations colliding with each other. If an animation is started while another one is mid way, the first one should be aborted and the second one should start from where the first one ended. If that doesn't happen with the globe rotation, just tag me and I can have a closer look.